tests/bindings/nodejs/HawkTracerNativeClient.test.js

```diff
@@ -85,7 +85,7 @@ describe("3. Start native HawkTracerClient", () => {
         const source = require('path').join(__dirname, 'test.htdump');
         // noinspection JSValidateTypes
         hawkTracerClient = new HawkTracerNativeClient(source);
-        expect(hawkTracerClient.start()).toBe(true);
+        expect(hawkTracerClient.start()).toEqual(true);
```

Comment:
Does this check the resolved value? Even so, it should check for `hawktracerClient`.

Reply:
I don't think this is correct – the HawkTracerNativeClient returns true/false, not a promise (it's the C++ implementation)

Reply:
Ah you're right.